examples/models/file/image_url.py

```diff
@@ -33,7 +33,7 @@
 image2 = ImageURL(url="url", x="x2", y="y2", w=20, h=20, anchor="top_left")
 plot.add_glyph(source, image2)
 
-image3 = ImageURL(url=dict(value=url), x=200, y=-100, anchor="bottom_right")
+image3 = ImageURL(url=dict(value=url), x=200, y=-100, w=50, h=50, angle=np.pi, anchor="bottom_right")
```

Comment:
>This may be a regression in ImageURL glyph. 

I don't think so. `ImageURL`s that don't have 'w' and 'h' set don't show up in 0.12.4 either.
<img width="400" alt="screen shot 2017-03-31 at 5 20 01 pm" src="https://cloud.githubusercontent.com/assets/4806877/24569908/8ce4190e-1636-11e7-8121-3d97c8377326.png">

Python version      :  3.5.3 
Bokeh version       :  0.12.4